frontend/sw360-portlet/src/main/webapp/css/sw360.css

```diff
@@ -326,3 +326,10 @@ input[readonly].clickable {
     word-break: break-all;
     font-family: monospace;
 }
+
+.formatSelect {
+   width: 20em !important;
```

Comment:
This should be solved by reordering the style sheets in the HTML section. One should not have to use `!important`!